addons/website/models/website.py

```diff
@@ -192,6 +192,14 @@ def _compute_menu(self):
     def _get_menu_ids(self):
         return self.env['website.menu'].search([('website_id', '=', self.id)]).ids
 
+    @tools.ormcache('self.env.uid', 'self.id')
+    def has_record_url_in_menu(self):
```

Comment:
Can you name it `is_menu_cache_disabled()` or something?
It would help if we need to add new stuff in this method like a fix later or something.

In master, we will add code here, I created this task which I just thought about reviewing this PR: https://www.odoo.com/web#menu_id=4720&cids=1&action=333&active_id=974&model=project.task&view_type=form&id=3800830
We can benefit from this diff to add back groups on menu at no cost